hdtv/rootext/fit/PolyBg.cc

```diff
@@ -37,9 +37,7 @@
 namespace HDTV {
 namespace Fit {
 
-PolyBg::PolyBg(int nParams,
-               const Option<bool> integrate,
-               const Option<std::string> likelihood) {
+PolyBg::PolyBg(int nParams, const Option<bool> integrate, const Option<std::string> likelihood) {
```

Comment:
Codacy found an issue: [Function parameter 'likelihood' should be passed by const reference.](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)